# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../quagga/linux-2.6.18.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./lib/zebra.h.orig	2006-10-11 01:31:42.000000000 +0200
+++ ./lib/zebra.h	2006-10-11 01:46:47.000000000 +0200
@@ -97,7 +97,9 @@
 #endif /* HAVE_LINUX_VERSION_H */
 
 #ifdef HAVE_ASM_TYPES_H
+#undef __STRICT_ANSI__
 #include <asm/types.h>
+#define __STRICT_ANSI__
 #endif /* HAVE_ASM_TYPES_H */
 
 /* misc include group */
